src/OpenTelemetry/Metrics/AggregationType.cs

```diff
@@ -56,21 +56,21 @@ internal enum AggregationType
         /// <summary>
         /// Histogram with sum, count, buckets.
         /// </summary>
-        Histogram = 6,
+        HistogramWithBuckets = 6,
```

Comment:
Renaming suggestion:

**NonBucketedHistogram** - sum and count
**BucketedHistogram** - sum and count + buckets
**NonBucketedHistogramWithMinMax** - sum and count + Min/Max
**BucketedHistogramWithMinMax**- sum and count + Min/Max + buckets

Reply:
Starting everything with Histogram is good for auto-completion as well... "non-bucketed " sounds weird, as buckets are anyway optional!
![image](https://user-images.githubusercontent.com/5232798/197895792-f5ac5b3a-4ff6-4c23-8c93-cc23d140b334.png)

Reply:
Auto-complete works with substrings as well.

> "non-bucketed " sounds weird, as buckets are anyway optional
I would say "non-bucketed" explicitly tells you that the buckets option was not enabled.